testdata/src/initialism/initialism.go

```diff
@@ -6,69 +6,69 @@ import "net/http"
 func _() {
 	h := http.Header{}
 
-	h.Get("A-IM")
-	h.Get("Accept-CH")
-	h.Get("ALPN")
-	h.Get("AMP-Cache-Transform")
-	h.Get("C-PEP")
-	h.Get("C-PEP-Info")
-	h.Get("Cal-Managed-ID")
-	h.Get("CalDAV-Timezones")
-	h.Get("CDN-Cache-Control")
-	h.Get("CDN-Loop")
-	h.Get("Content-ID")
-	h.Get("Content-MD5")
-	h.Get("DASL")
-	h.Get("DAV")
-	h.Get("Differential-ID")
-	h.Get("DNT")
-	h.Get("DPoP")
-	h.Get("DPoP-Nonce")
-	h.Get("EDIINT-Features")
-	h.Get("ETag")
-	h.Get("Expect-CT")
-	h.Get("GetProfile")
-	h.Get("HTTP2-Settings")
-	h.Get("IM")
-	h.Get("Include-Referred-Token-Binding-ID")
-	h.Get("Last-Event-ID")
-	h.Get("MIME-Version")
-	h.Get("NEL")
-	h.Get("OData-EntityId")
-	h.Get("OData-Isolation")
-	h.Get("OData-MaxVersion")
-	h.Get("OData-Version")
-	h.Get("Optional-WWW-Authenticate")
-	h.Get("OSCORE")
-	h.Get("OSLC-Core-Version")
-	h.Get("P3P")
-	h.Get("PEP")
-	h.Get("PEP-Info")
-	h.Get("PICS-Label")
-	h.Get("ProfileObject")
-	h.Get("Repeatability-Client-ID")
-	h.Get("Repeatability-Request-ID")
-	h.Get("Sec-GPC")
-	h.Get("Sec-WebSocket-Accept")
-	h.Get("Sec-WebSocket-Extensions")
-	h.Get("Sec-WebSocket-Key")
-	h.Get("Sec-WebSocket-Protocol")
-	h.Get("Sec-WebSocket-Version")
-	h.Get("SetProfile")
-	h.Get("SLUG")
-	h.Get("SoapAction")
-	h.Get("Status-URI")
-	h.Get("TCN")
-	h.Get("TE")
-	h.Get("TTL")
-	h.Get("URI")
-	h.Get("WWW-Authenticate")
-	h.Get("X-Correlation-ID")
-	h.Get("X-DNS-Prefetch-Control")
-	h.Get("X-Real-IP")
-	h.Get("X-Request-ID")
-	h.Get("X-UA-Compatible")
-	h.Get("X-WebKit-CSP")
-	h.Get("X-XSS")
-	h.Get("X-XSS-Protection")
+	h.Get("A-Im")                              // want `initialism header "A-Im", instead use: "A-IM"`
```

Comment:
This error message is unclear.

It's the same with non-canonical and the others.

Maybe something like


```suggestion
	h.Get("A-Im")                              // want `"A-Im" header should be "A-IM" (initialism)`
```

Or canonical, I won't add the negation to canonical. Because initialism and canonical are descriptive, when it's written without the expected case.